src/notebook/notebook/actions.ts

```diff
@@ -826,7 +826,12 @@ namespace Private {
       break;
     case 'code':
       if (kernel) {
-        return (widget as CodeCellWidget).execute(kernel);
+        return (widget as CodeCellWidget).execute(kernel).then(reply => {
+          if (reply)
+            return reply.content.status === 'ok';
+          else
+            return true;
```

Comment:
athough this could be a one-liner: `return reply ? reply.content.status === 'ok' : true;`